### PR TITLE
ansible-galaxy, az: update Italian translations

### PR DESCRIPTION
**Pull request by Astr0surf3r**
fix 2 files part of the [#101](https://github.com/tldr-pages/tldr-maintenance/issues/101) issue for the IT translations 

da-caricare

- [x] The page(s) are in the correct platform directories: `common`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages.it/common/az.md
[the remark this answers is not in the document]
Comment (Astr0surf3r) [reply pages.it/common/az.md:1]: @sebastiaanspeck done check it out [15a61e8](https://github.com/tldr-pages/tldr/pull/14513/commits/15a61e8ec5a9e66a137c9d3f3a1fe16562b4639b)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by Astr0surf3r** [comment]
> LGTM, Thanks for your contribution.

thank you

**Review by karthik-script — APPROVED** [review]

File: pages.it/common/az.md [hunk 1]
```diff
@@ -2,28 +2,36 @@
...
-- Gestisci le risorse della rete di Azure:
+- Gestisce le risorse della rete di Azure:
 
 `az network`
+
+- Avvia la modalità interattiva:
+
+`az interactive`
+
+- Mostra la descrizioen dei comandi:
```
Comment (gutjuri) [inline pages.it/common/az.md:35]:
```suggestion
- Mostra la descrizioni dei comandi:
```

**Review by gutjuri — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

File: pages.it/common/az.md (hunk 1 above)
Comment (Astr0surf3r) [reply pages.it/common/az.md:35]: @gutjuri thanks a lot for your message I have just update the last commit

**Review by mebeim — CHANGES_REQUESTED** [review]
Thanks for the contribution @Astr0surf3r. The `az` page looks nice, but I think there are a lot of minor translation issues with the `ansible-galaxy` page (see below). You can either apply those here or I can do it for you, just let me know. Thanks again!

File: pages.it/common/az.md [hunk 2]
```diff
@@ -2,28 +2,36 @@
...
-- Gestisci le risorse della rete di Azure:
+- Gestisce le risorse della rete di Azure:
 
 `az network`
+
+- Avvia la modalità interattiva:
+
+`az interactive`
+
+- Mostra le descrizioni dei comandi:
```
[the remark this answers is not in the document]
Comment (Astr0surf3r) [reply pages.it/common/az.md:35]: @mebeim thanks so much for your tweaks! Perhaps we need a guideline for the Italian translation of tldr—maybe a [gist](https://gist.github.com/)? What do you think?"

File: pages.it/common/az.md [hunk 3]
```diff
@@ -8,22 +8,30 @@
 
 `az login`
 
-- Gestisci la tua iscrizione ad Azure:
+- Gestisce il tuo abbonamento ad Azure:
```
Comment (mebeim) [inline pages.it/common/az.md:11]:
```suggestion
- Gestisci il tuo abbonamento ad Azure:
```

File: pages.it/common/az.md [hunk 4]
```diff
@@ -8,22 +8,30 @@
...
 
 `az disk list`
 
-- Elenca tutte le virtual machine Azure:
+- Elenca tutte le macchine virtuali di Azure:
 
 `az vm list`
 
-- Gestisci i servizi Azure Kubernetes:
+- Gestisce i servizi Azure Kubernetes:
```
Comment (mebeim) [inline pages.it/common/az.md:23]:
```suggestion
- Gestisci i servizi Azure Kubernetes:
```

File: pages.it/common/az.md [hunk 5]
```diff
@@ -8,22 +8,30 @@
...
 
 `az aks`
 
-- Gestisci le risorse della rete di Azure:
+- Gestisce le risorse della rete di Azure:
```
Comment (mebeim) [inline pages.it/common/az.md:27]:
```suggestion
- Gestisci le risorse della rete di Azure:
```

File: pages.it/common/ansible-galaxy.md [hunk 6]
```diff
@@ -1,32 +1,32 @@
 # ansible-galaxy
 
-> Crea e gestisci ruoli di Ansible.
+> Esegui varie operazioni inerenti i Ruoli e le Collezioni in Ansible.
```
Comment (mebeim) [inline pages.it/common/ansible-galaxy.md:3]:
```suggestion
> Esegui varie operazioni inerenti ai Ruoli e alle Collezioni in Ansible.
```

File: pages.it/common/ansible-galaxy.md [hunk 7]
```diff
@@ -1,32 +1,32 @@
...
+`ansible-galaxy role {{install|remove}} {{nome_ruolo1 nome_ruolo2 ...}}`
 
-- Cerca un determinato ruolo:
+- Crea un nuovo ruolo:
 
-`ansible-galaxy search {{nome_ruolo}}`
+`ansible-galaxy role init {{nome_ruolo}}`
 
-- Crea un nuovo ruolo:
+- Ottieni informazioni inerenti un ruolo:
```
Comment (mebeim) [inline pages.it/common/ansible-galaxy.md:22]:
```suggestion
- Ottieni informazioni inerenti a un ruolo:
```

**Review by mebeim — CHANGES_REQUESTED** [review]
Hey @Astr0surf3r thanks for the changes, but I think I have missed a few of them in the last review. There are still some instances where third person is used (see below). I can also apply these for you if you want, LMK.

**Review by Magrid0 — APPROVED** [review]
Looks good to me

**Comment by Magrid0** [comment]
> What about the comments from @mebeim?

they’re good to go, sorry for the late reply.

**Comment by mebeim** [comment]
All right, this has been sitting here for a while. I just applied the few remaining suggestions myself. Thanks again!

**Pull request merged**